src/javascript/_common/base/currency_base.js

```diff
@@ -78,11 +78,24 @@ const CryptoConfig = (() => {
 
 const getMinWithdrawal = currency => (isCryptocurrency(currency) ? (getPropertyValue(CryptoConfig.get(), [currency, 'min_withdrawal']) || 0.002) : 1);
 
-// returns in a string format, e.g. '0.00000001'
-const getMinTransfer = currency => {
-    const min_transfer = getPropertyValue(currencies_config, [currency, 'transfer_between_accounts', 'limits', 'min']) || getMinWithdrawal(currency);
+/**
+ * Returns the transfer limits for the account.
+ * @param currency
+ * @param which
+ * @returns {min: numeric, ?max: numeric}
+ */
+const getTransferLimits = (currency, which) => {
+    const transfer_limits = getPropertyValue(currencies_config, [currency, 'transfer_between_accounts', 'limits']) || getMinWithdrawal(currency);
     const decimals     = getDecimalPlaces(currency);
-    return min_transfer.toFixed(decimals); // we need toFixed() so that it doesn't display in scientific notation, e.g. 1e-8 for currencies with 8 decimal places
+
+    switch (which) {
+        case 'max':
+            return transfer_limits.max ? transfer_limits.max.toFixed(decimals) : undefined;
+        case 'all':
+            return transfer_limits;
```

Comment:
nitpicking about the fallback option again, if we fallback to getMinWithdrawal for `transfer_limits` we won't have and object with min and max, but otherwise we will. I suggest removing the `all` case altogether since it's not used anywhere, or handling the fallback better if you want to keep it.

Reply:
I want to save the behaviour of giving out both of the limits for future. So I have added a better failsafe for it.